.github/workflows/release.yml

```diff
@@ -489,12 +489,9 @@ jobs:
           make release-cli
 
       - name: CLI - create release with binaries
-        uses: softprops/action-gh-release@v1
+        uses: softprops/action-gh-release@v2
         with:
           draft: true
           files: |
             dist/*
-        env:
-          GITHUB_TOKEN: ${{ github.token }}
-
```

Comment:
The `env` key is deprecated, see https://github.com/softprops/action-gh-release?tab=readme-ov-file#environment-variables
I don't think it's needed anymore as the `token` key defaults to `${{ github.token }}`, see https://github.com/softprops/action-gh-release?tab=readme-ov-file#inputs